src/main/webapp/app/exercises/modeling/assess/modeling-assessment-editor/modeling-assessment-dashboard.component.ts

```diff
@@ -143,7 +143,7 @@ export class ModelingAssessmentDashboardComponent implements OnInit, OnDestroy {
                 this.assessedSubmissions = this.submissions.filter((submission) => {
                     const result = getLatestSubmissionResult(submission);
                     setLatestSubmissionResult(submission, result);
-                    return !!result;
+                    return result?.rated;
```

Comment:
Is there a specific reason for this change?

Reply:
`return !!result;` returns the submissions (whit or without grade). That is the reason why the progress message shows 100% assessed submissions (screenshot 1)  but just 1 / 3 are already assessed. 
To get only the graded submissions (when a tutor has already submitted a grade) I use `return result?.rated;`

Reply:
Okay, the problem was then that result was always defined for both graded and ungraded exercises. The reason was not described in the PR description, therefore I asked.